dist/images/ovnkube.sh

```diff
@@ -996,6 +999,11 @@ local-nb-ovsdb() {
   ovn-nbctl set NB_Global . name=${K8S_NODE}
   ovn-nbctl set NB_Global . options:name=${K8S_NODE}
 
+ [[ "true" == "${ENABLE_IPSEC}" ]] && {
+    ovn-nbctl set nb_global . ipsec=true
```

Comment:
note to reviewers; set -e is enable so this would fail and exit if theres an issue setting this.